### DISCUSSION TITLE
How to read energyyearwall.txt

### DISCUSSION THREAD

**Discussion by albmorsam** [UMEP-dev/UMEP#624] [Q&A]
[Energyyearwall.txt](https://github.com/user-attachments/files/15586533/Energyyearwall.txt)
Hi,

Any tutorial about how to read energyyearwall.txt? how can I identify the col and row for the building's façade highlighted in yellow? this case is the one from Gotherborg taken as an example. 

thanks in advance

Regards

![image](https://github.com/UMEP-dev/UMEP-processing/assets/125660123/6eb67375-2e10-4d66-aa71-d1b2cc65b09d)

**Comment by biglimp** [comment]
It has been on our TODO-list for some time now to fix a nicer output from SEBE but we never found the time... We are currently working on an updte of the SOLWEIG-model that might help us to fix a nicer output also from SEBE. 

To find the rows and columns you need to convert from geographical coordinates to rows and columns in energyyearwall.txt. I think the easiest way at the moment would be the following:

1. Convert walls.tif to a point vector layer and extract x and y coordinates to the attribute table
2. Add two new tables where you calculate row and col by subtracting the higher left corner of walls.tif with the coordinates of the points (absolute values). You probably need to round all values.
3. Select all points of the building you are interested in and here you have the rows and cols that you want.
